proto/spacemesh/mesh_types.proto

```diff
@@ -3,7 +3,7 @@ package spacemesh;
 
 import "spacemesh/types.proto";
 
-enum AccountMeshDataFlags {
```

Comment:
I think this should be plural... one bitfield contains many flags

Reply:
I had a look at the naming pattern of enums in the proto3 google docs examples and they all seem to be singular - agree we should use 1 style and be consistent. I think that right now all enums are singular, or at least should be.